doi-transformer/src/test/java/no/unit/nva/doi/transformer/CrossRefConverterTest.java

```diff
@@ -689,7 +762,7 @@ private CrossRefDocument sampleCrossRefDocumentWithBasicMetadata() {
         CrossRefDocument crossRefDocument = new CrossRefDocument();
         setAuthor(crossRefDocument);
         crossRefDocument.setPublisher(SAMPLE_PUBLISHER);
-        crossRefDocument.setTitle(List.of(SAMPLE_DOCUMENT_TITLE));
+        crossRefDocument.setTitle(List.of(SAMPLE_DOCUMENT_TITLE, ALTERNATIVE_TITLE));
```

Comment:
Should this be ".setTitles" since its a list

Reply:
The CrossrefDocument is a Java represenation of the response we get from Crossref. The field names represent the actual names in the json object. While it is true that we should rename  the array fields of the json to "titles", "alternativeTitles", "originalTitles" etc. it should be a PR on its own.